app/views/catalog/_document.html.erb

```diff
@@ -0,0 +1,5 @@
+<%# Overrides TRLN Argon partial to help with semantic markup %>
```

Comment:
Looks like the only things different from `trln_argon` are the div -> section tag and the addition of `aria-label` (a clever alternative to rearranging the markup order!). Hopefully those would be welcome or at least unobtrusive changes for all of TRLN. So if it's possible, it'd be better to change at the engine level to spare us from having a local custom copy of this partial. Could also be addressed at a future juncture if this'd hold up rolling out these important a11y changes.